### PR TITLE
Blog for JBDS-8.1.0.Beta1

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

File: blog/2015-03-05-JBDS-8.1.0.Beta1-for-luna.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+We are ready with link:/downloads/devstudio/luna/8.1.0.Beta1.html[Red Hat JBoss Developer Studio 8.1.0 Beta1] and link:/downloads/jbosstools/luna/4.2.3.Beta1.html[JBoss Tools 4.2.3 Beta1] for Eclipse Luna SR2.
+
+image::images/jbosstools-jbdevstudio-blog-header.png[]
+
+== Updating from JBoss Tools 4.2.x or JBoss Developer Studio 8.0.x
+
+If you have JBoss Tools 4.2.x or JBoss Developer Studio 8.0.x already installed, just run:
+
+    Help > Check for updates
```
Comment (maxandersen) [inline blog/2015-03-05-JBDS-8.1.0.Beta1-for-luna.adoc]: this wont work for beta's. 

here you need to use the development updatesite explicitly.

Comment (alexeykazakov) [inline blog/2015-03-05-JBDS-8.1.0.Beta1-for-luna.adoc]: Right. I missed that. Will fix it.

File: blog/2015-03-05-JBDS-8.1.0.Beta1-for-luna.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    java -jar jboss-devstudio-{version}-installer-{standalone|eap}.jar
+
+JBoss Tools or JBoss Developer Studio Bring-Your-Own-Eclipse (BYOE) requires a bit more:
+
+This release requires at least Eclipse 4.4 (Luna) but we recommend using the
+http://www.eclipse.org/downloads/packages/eclipse-ide-java-ee-developers/lunasr2[Eclipse Luna SR2 Java EE Bundle] since then you get most of the dependencies preinstalled with Java EE Bundle and SR2 release includes some critical fixes.
+
+Once you have installed or if you have JBoss Tools 4.2.x already installed, you can use our update site directly for JBoss Tools:
+
+    http://download.jboss.org/jbosstools/updates/stable/luna/
```
Comment (maxandersen) [inline blog/2015-03-05-JBDS-8.1.0.Beta1-for-luna.adoc]: /updates/development/luna

**Comment by alexeykazakov** [comment]
Merged.

**Pull request closed without merging**